packages/gatsby-theme/src/components/Contact/Contact.stories.mdx

```diff
@@ -33,32 +33,31 @@ export const contactInfo = [
 
 ## Props
 
-<Props of={Contact} />
+<ArgsTable of={Contact} />
```

Comment:
Since this is a compound component, we could maybe add the information of the subcomponents.
https://github.com/storybookjs/storybook/issues/9444